corehq/apps/case_importer/static/case_importer/js/import_history.js

```diff
@@ -19,7 +19,25 @@ hqDefine('case_importer/js/import_history.js', function () {
             // this prevents some jumpiness when not necessary
             // and is particularly bad if you're in the middle of editing a comment
             return !(_.chain(self.case_uploads()).pluck('upload_id').isEqual(_(data).pluck('upload_id')).value() &&
-                _.chain(self.case_uploads()).pluck('task_status').isEqual(_(data).pluck('task_status')).value());
+                _.chain(self.case_uploads()).pluck('task_status').map(function (task_status) {return task_status()})
+                    .isEqual(_(data).pluck('task_status').map(function (task_status) {return task_status()})).value());
+        };
+        self.updateCaseUploads = function (data) {
```

Comment:
The idea in my head at least is that this function is _conceptually_ equivalent to just `self.case_uploads()` (which it calls when it can't do the easy optimization), but one that has better behavior in terms of updating the DOM in place instead of replacing it entirely.

That's where the name comes from, but I agree the name leaves the purpose behind its existence a bit of a mystery. Maybe something like `self.updateCaseUploadsWithMinimalEffectOnDOM` would be clearer, which may be worth the verbosity?